### PR TITLE
EOS-23885: dgread not working with motr utils, while 1 node is down

### PR DESCRIPTION
**Pull request by yatin-mahajan**
Problem:
  In case of node failures motr utils(m0cat) failed to establish connection
  on endpoints of failed node, which actually fails the session_validate()
  checks with EINVAL in target_ioreq_iofops_prepare(), that fails this
  reading operation.

Solution:
  Allowed target_ioreq_iofop_prepare() to process further event
  though session is INVALID, so that it can proceed further for
  degraded operation.

Signed-off-by: Yatin Mahajan <yatin.mahajan@seagate.com>

### CODE REVIEW

**Comment by imvenkip** [comment]
@yatin-mahajan  Could you update testing output?

File: motr/io_nw_xfer.c [hunk 1]
```diff
@@ -863,7 +863,7 @@ static int target_ioreq_iofops_prepare(struct target_ioreq *ti,
 	M0_PRE(M0_IN(filter, (PA_DATA, PA_PARITY)));
 
 	rc = m0_rpc_session_validate(ti->ti_session);
-	if (rc != 0 && rc != -ECANCELED)
+	if (rc != 0 && (!M0_IN(rc,( -ECANCELED, -EINVAL))))
```
Comment (imvenkip) [inline motr/io_nw_xfer.c:866]: [cosmetic] Add space after rc and remove space before -ECANCELED.

Comment (yatin-mahajan) [reply motr/io_nw_xfer.c]: Done

**Review by imvenkip — APPROVED** [review]
Looks good to me.

**Review by yeshpal-jain-seagate — APPROVED** [review]

**Review by mehjoshi — APPROVED** [review]

**Pull request merged**